packages/react-examples/src/react-experiments/FloatingPeopleSuggestions/FloatingPeopleSuggestions.Example.tsx

```diff
@@ -88,7 +88,7 @@ export const FloatingPeopleSuggestionsExample = (): JSX.Element => {
       <FloatingPeopleSuggestions
         suggestions={[...peopleSuggestions]}
         isSuggestionsVisible={true}
-        targetElement={null}
+        targetElement={{ left: 20, top: 20 }}
```

Comment:
>{{ left: 20, top: 20 } [](http://example.com/codeflow?start=22&length=22)

Making a note so I remember to look something up.

I was thinking "wait, this is an object, and a new object is created every render, and strict equality checks will mean this is 'different' every time, so maybe this causes extra renders?? So maybe use memoization so it's the same object reference every render?"

BUT even if this is memoized, it's still an object... oh, got it, but it's the SAME object every time which means it passes strict equality check if we use memoization. That said, I'm not sure if that means we need memoization here... I want to double check some assumptions before suggesting a change here.